operations/workplace/people/people-compliance.md

```diff
@@ -27,3 +27,62 @@ US Equal Employment Opportunity Commission
 * [Know Your Rights: Workplace Discrimination is Illegal (English)](https://www.eeoc.gov/know-your-rights-workplace-discrimination-illegal-poster)
 * [Know Your Rights: Workplace Discrimination is Illegal (Spanish)](https://www.eeoc.gov/es/conozca-sus-derechos-la-discriminacion-en-el-lugar-de-trabajo-es-ilegal)
 * [Family and Medical Leave Act (FMLA) Poster (US)](https://www.dol.gov/sites/dolgov/files/WHD/legacy/files/fmlaen.pdf)
+
+### Notification of Employee Rights Under the National Labor Relations Act (NLRA)
+
+* [Employee Rights under the NLRA Notice](https://www.dol.gov/sites/dolgov/files/olms/regs/compliance/eo_posters/employeerightsposter11x17_2019final.pdf)
+
+### Whistleblower Protection Notices 
+
+Notices for US States
+
+* [Carlifornia Whistleblower Protection Notice](https://www.dir.ca.gov/dlse/whistleblowersnotice.pdf)
+* [Colorado Whistleblower Protection Notice](https://cdle.colorado.gov/sites/cdle/files/%5BCLEAN%20June%201%2C%202022%5D%20Poster%2C%20Paid%20Leave%20%26%20Whistleblower.pdf)
+* [Hawaii Whistleblower Protection Notice](https://labor.hawaii.gov/wp-content/uploads/2013/07/Whistle-Blower-Poster.pdf)
+* [Maine Whistleblower Protection Notice](https://www.maine.gov/labor/docs/2022/posters/whistleblower/whistleblowerprotection_English.pdf)
+* [New Hampshire Whistleblower Protection Notice](https://www.nh.gov/labor/documents/whistleblower-poster.pdf)
+* [New Jersey Whistleblower Protection Notice](https://www.nj.gov/labor/wageandhour/assets/PDFs/Employer%20Poster%20Packet/CEPA270.1.pdf)
+* [New York Whistleblower Protection Notice](https://www.nysdental.org/docs/librariesprovider37/default-document-library/new-york-state-department-of-labor-required-employer-whistleblower-notice.pdf?sfvrsn=f7b4f08a_0)
+* [Rhode Island Whistleblower Protection Notice](http://webserver.rilin.state.ri.us/Statutes/TITLE28/28-50/28-50-3.htm)
+
+### US States Sick Leave Policies 
+
+Sick Leave Posters for US states/Cities
+
+* [Arizona Sick Leave Poster](https://www.azica.gov/sites/default/files/AZ%20Earned%20Paid%20Sick%20Time%20Poster%202017.pdf)
+* [Carlifornia Sick Leave Poster](https://www.dir.ca.gov/dlse/publications/paid_sick_days_poster_template_(11_2014).pdf)
+* [Berkeley(Carlifornia) Sick Leave Poster](https://employees.aleroninc.com/wp-content/uploads/2018/07/berkley-ca-paid-sick-leave-policy-and-poster.pdf)
```

Comment:
Hi @yomiadetutu1! Great job with this PR. I recommend adding an extra space between city names and states where needed in this list. For example, on this line, an extra space is needed between ``Berkeley`` and ``(California)``. Also note that ``Carlifornia`` should be ``California`` throughout.